NitroxServer/Communication/Packets/Processors/UpdateBaseProcessor.cs

```diff
@@ -0,0 +1,23 @@
+using NitroxModel.Packets;
+using NitroxServer.GameLogic;
+using NitroxServer.GameLogic.Bases;
+
+namespace NitroxServer.Communication.Packets.Processors;
+
+public class UpdateBaseProcessor : BuildingProcessor<UpdateBase>
+{
+    public UpdateBaseProcessor(BuildingManager buildingManager, PlayerManager playerManager) : base(buildingManager, playerManager) { }
+
+    public override void Process(UpdateBase packet, Player player)
+    {
+        if (buildingManager.UpdateBase(player, packet, out int operationId))
+        {
+            packet.BaseData = null;
+            packet.BuiltPieceEntity = null;
+            packet.UpdatedChildren = null;
+            packet.UpdatedMoonpools = null;
+            packet.UpdatedMapRooms = null;
```

Comment:
Would explain here why we set the fields to null too. Maybe even move this to a packet.Reset() or Deflate(), etc,